spython/utils/terminal.py

```diff
@@ -140,9 +146,14 @@ def run_command(cmd,
        capture: if True, don't set stdout and have it go to console. This
                 option can print a progress bar, but won't return the lines
                 as output.
+       sudo_options: string or list of strings that will be passed as options to sudo
     '''
 
-    if sudo:
+    if sudo and sudo_options is not None:
```

Comment:
I'm seeing duplicate code :)

Reply:
In the same way that we have self._init_command() perhaps there could be a function to handle sudo options, that way if there is a bug fix or similar, we don't have to do it in two places. And do you see sudo options being useful in other functions, aside from build?

Reply:
Duplicate code replaced with a function

Reply:
I am not sure about sudo usage elsewhere, I had never used `spython` before this afternoon so not familiar with it really :smile:

Reply:
I assume I need to bump the version and update the changelog?

Anything else?

Reply:
Welcome to the library! :D Glad to have you here too :)